src/lucky_record/form.cr

```diff
@@ -167,6 +167,18 @@ abstract class LuckyRecord::Form(T)
     end
   end
 
+  def save! : T
+    if save
+      record.as(T)
```

Comment:
I think the idea here was to tell the compiler that it is not nil right? If so you can call `not_nil!` on it